### PR TITLE
Note that replications slots are not restored.

### PR DESCRIPTION
**Pull request by jreidthompson**
Update documentation and help to note that replication slots are not restored and why. Reference PostgreSQL documentation.

https://github.com/pgbackrest/pgbackrest/projects/2#card-74790384
https://github.com/pgbackrest/pgbackrest/issues/1585

### CODE REVIEW

**Review by pgstef — APPROVED** [review]
Hey,
To me, this sounds simple enough as explanation to avoid duplicating what's in PG docs.
Reading the docs should always be a good idea and with the link (+ subsection), the paragraph about replication slots is easy to find.
So, +1 for me :-)

File: src/build/help/help.xml [hunk 1]
```diff
@@ -2112,6 +2112,8 @@
...
 
                     <p>When PITR of <br-option>{[dash]}-type=time</br-option> is specified, then the target time must be specified with the <br-option>{[dash]}-target</br-option> option. If a backup is not specified via the <br-option>{[dash]}-set</br-option> option, then the configured repositories will be checked, in order, for a backup that contains the requested time. If no backup can be found, the latest backup from the first repository containing backups will be used. For other types of PITR, e.g. <id>xid</id>, the <br-option>{[dash]}-set</br-option> option must be provided if the target is prior to the latest backup. See <link page="user-guide" section="/pitr">Point-in-Time Recovery</link> for more details and examples.</p>
+
+                    <p>Replication slots are not included per recommendation of the <postgres/> documentation. See the <postgres/> documentation: <link url="https://www.postgresql.org/docs/current/continuous-archiving.html#BACKUP-LOWLEVEL-BASE-BACKUP">Making a Base Backup Using the Low Level API</link> (subsection Backing Up The Data Directory) for more information.</p>
```
Comment (dwsteele) [inline src/build/help/help.xml:2116]: Perhaps it would be better to link directly to the subsection we want (#BACKUP-LOWLEVEL-BASE-BACKUP-DATA)?

Comment (jreidthompson) [reply src/build/help/help.xml]: resolved 476854f9b5fc043976d9bc6f387ddaa87fcbe79c 
Thanks, I had looked for but failed to determine how to link that subsection.

Comment (dwsteele) [reply src/build/help/help.xml]: I do it by searching for the section title in the page source, then finding the link nearby.

**Review by dwsteele — APPROVED** [review]
Some edits in 2224b052a. If those look good this is ready to go.

**Review by jreidthompson — COMMENTED** [review]
looks good to me

**Pull request merged**